<ui version="4.0" >
 <author>Hans Meine &lt;hans_meine@gmx.net></author>
 <class>Colorize</class>
 <widget class="QMainWindow" name="Colorize" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Colorize</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QImageViewer" name="imageViewer" />
    </item>
    <item>
     <widget class="ColorMapEditor" name="cme" />
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>&amp;Gamma:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>gammaSlider</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gammaSlider" >
        <property name="minimum" >
         <number>-15</number>
        </property>
        <property name="maximum" >
         <number>15</number>
        </property>
        <property name="pageStep" >
         <number>5</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="gammaSpinBox" />
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QImageViewer</class>
   <extends>QWidget</extends>
   <header>VigraQt/qimageviewer.hxx</header>
  </customwidget>
  <customwidget>
   <class>ColorMapEditor</class>
   <extends>QWidget</extends>
   <header>VigraQt/cmeditor.hxx</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >VigraQt/qimageviewer.hxx</include>
 </includes>
 <resources/>
 <connections/>
</ui>
